sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -966,6 +976,7 @@ def process_bundle(self, instruction_id):
         output_stream = self.timer_data_channel.output_timer_stream(
             instruction_id, transform_id, timer_family_id)
         timer_info.output_stream = output_stream
+        # FIXME: how do we know that self.ops[transform_id] is a DoOperation?
         self.ops[transform_id].add_timer_info(timer_family_id, timer_info)
```

Comment:
`BundleProcessor.ops` is `OrderedDict[str, operations.Operation]`.  Should it be `OrderedDict[str, operations.DoOperation]`?  Or do the transform ids in `BundleProcessor.timers_info` all point to `DoOperations`?